### DISCUSSION TITLE
Get Bits Per Sample info with track

### DISCUSSION THREAD

**Discussion by Aionir** [Zeugma440/atldotnet#119] [Q&A] [answered]
I don't see a way to get the bits per sample  for the specified audio track. Bitrate and Samplerate are provided, however.

Is there a way to get this info or is it not supported?

**Comment by Zeugma440** [comment, marked as answer]
`Track` doesn't give you bits per sample directly, but you can calculate it with any of these two methods :

A - `1000 * Track.Bitrate  / Track.SampleRate`

B - `8 * Track.TechnicalInfo.AudioDataSize / (Track.Duration * Track.SampleRate)`

**Comment by Zeugma440** [comment]
BIt depth (bits per sample) is now directly accessible through `Track.BitDepth` in today's v4.15
